package.json

```diff
@@ -18,48 +18,47 @@
     "url": "https://github.com/dojo/cli-test-intern.git"
   },
   "scripts": {
+    "build:static:dev": "cpx \"{src,tests}/**/*.{d.ts,json,test}\" dist/dev",
+    "build:static:release": "cpx \"src/**/*.{d.ts,json}\" dist/release",
+    "build:cjs": "tsc",
+    "build": "concurrently \"npm:build:*\" && dojo-package",
+    "clean": "rimraf dist coverage",
+    "lint": "tslint -p .",
     "precommit": "lint-staged",
     "prettier": "prettier --write 'src/**/*.ts' 'tests/**/*.ts'",
-    "test": "grunt test"
+    "test": "npm run build && intern",
```

Comment:
We should add `prettier -l {src,tests}/**/*.ts` to test in order to fail the build if files don't comply. It used to be possible to not run precommit if the user already had a git precommit script. Don't know if this has been fixed in husky.